ps_emailsubscription.php

```diff
@@ -1214,4 +1215,41 @@ private function getConditionFixtures($lang)
             $this->trans('You may unsubscribe at any moment. For that purpose, please find our contact info in the legal notice.', array(), 'Modules.Emailsubscription.Shop', $locale)
         ;
     }
+
+    /**
+     * hook display AdminCustomers form
```

Comment:
This doesn't really add much information, please write a more precise description.